app/assets/javascripts/oxalis/view/right-menu/comment_tab/tree_comment_list.js

```diff
@@ -25,30 +26,51 @@ type State = {
 };
 
 class TreeCommentList extends React.PureComponent<TreeCommentListProps, State> {
+  treeDomElement: ?HTMLLIElement;
+  activeNodeHasComment: boolean;
   state = {
     collapsed: false,
   };
 
+  componentDidUpdate() {
+    this.ensureVisible();
+  }
+
+  ensureVisible() {
+    // Only scroll to this tree if this is the active tree and there is no active comment to scroll to
+    if (
+      this.props.tree.treeId === this.props.skeletonTracing.activeTreeId &&
+      !this.activeNodeHasComment
```

Comment:
Is `this.activeNodeHasComment` not check that could be done here instead of in the `render` function? Perhaps, to avoid slow downs, we only need to iterate over all comments if the we are indeed in the active tree.